requirements.txt

```diff
@@ -4,3 +4,5 @@ flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
 pep8-naming==0.13.2
 pytest==7.1.3
+
+player~=0.6.1
```

Comment:
why you need this?

Reply:
its a mistake by pycharm, i will fix it.